.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -184,6 +185,8 @@ jobs:
 
           CIBW_BUILD_VERBOSITY: 3
 
+          OMP_PROC_BIND: false
```

Comment:
This is just to avoid a warning that `OMP_PROC_BIND` is better set to `false` for unit tests.

Reply:
Isn't that warning exclusive to the Kokkos OMP backend? Since we only build serial Kokkos here, it shouldn't be an issue, no?

Reply:
Right, I'm testing locally with my default Kokkos which has OMP and I thought that was a generic warning. I think it doesn't hurt, but I can remove it until we distribute OMP-enabled Kokkos wheels. What do you prefer?

Reply:
I think we can remove it from here, since it is not appearing in our build/test process for Lightning. We can revisit later if we distribute OMP based Kokkos builds.

Reply:
Removed.